src/FirmwarePlugin/APM/ArduCopterFirmwarePlugin.h

```diff
@@ -59,6 +59,7 @@ class ArduCopterFirmwarePlugin : public APMFirmwarePlugin
 
     // Overrides from FirmwarePlugin
     void    guidedModeLand                      (Vehicle* vehicle) final;
+    void    guidedModeGotoLocation          (Vehicle* vehicle, const QGeoCoordinate& gotoCoord) override;
```

Comment:
In reality it's a bug that all of these aren't marked override like this new one. '''final''' prevents custom builds from overriding the QGC provided implementations which should be allowed.